src/components/buttons/AccountFormButton.tsx

```diff
@@ -31,6 +31,7 @@ export default function AccountFormButton({
 
   return (
     <>
+      {/* @ts-ignore */}
       <Modal
```

Comment:
Wasn't able to fix the error that appears with the new version of types/react... will re-visit at some point once time passes